src/components/Textarea/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { InputLayout, InputLayoutProps, Typography } from '..';
+import { Colors } from '../../styles';
+import { HTMLAttributes } from 'react';
+import cx from 'classnames';
+import styles from './Textarea.module.scss';
+
+export interface TextareaProps
+  extends HTMLAttributes<HTMLTextAreaElement>,
+    Omit<InputLayoutProps, 'children'> {
+  outlineColor?: Colors;
+  limit?: number;
+  value?: string;
+}
+
+function Textarea({
+  hideLabel,
+  label,
+  status,
+  className,
+  outlineColor,
+  limit,
+  ...props
+}: TextareaProps) {
+  const count = props.value?.length ?? 0;
+  const atLimit = limit ? count >= limit : false;
+
+  return (
+    <InputLayout
```

Comment:
I think that this component's status text might be too big. Double check with the design and confirm/fix it as needed pls while your at it